pipelines/main/launch_upload_jobs.yml

```diff
@@ -84,6 +88,11 @@ steps:
           export BUILDKITE_PLUGIN_CRYPTIC_BASE64_SIGNED_JOB_ID_SECRET
 
           export ALLOW_FAIL="true"
+          
+          # allow test-failures for non-release/tags so that nightly & PR builds
+          # can be uploaded to s3 even if tests are not passing
+          # (allowing for usage with juliaup)
+          export ALLOW_TEST_FAILURES=$(([[ "${BUILDKITE_BRANCH?}" == release-* ]] || [[ "${BUILDKITE_TAG:-}" == v* ]]) && echo "false" || echo "true")
```

Comment:
Same as above - would be nice to quickly test in CI (in a small GitHub Actions CI job).